packages/valory/skills/abstract_round_abci/models.py

```diff
@@ -541,11 +542,11 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
         self.url: str = self._ensure("url", kwargs, str)
         self.api_id: str = self._ensure("api_id", kwargs, str)
         self.method: str = self._ensure("method", kwargs, str)
-        self.headers: List[Tuple[str, str]] = self._ensure(
-            "headers", kwargs, List[Tuple[str, str]]
+        self.headers: Dict[str, str] = dict(
```

Comment:
why not OrderedDict twice?

Reply:
Well, I thought it would not make much sense to require an `OrderedDict` for all the HTTP requests with `python >=3.7`.